src/main/java/picard/sam/AbstractAlignmentMerger.java

```diff
@@ -409,7 +410,7 @@ public void mergeAlignment(final File referenceFasta) {
                         final SAMRecord matePrimary = isRead1 ? r2Primary : r1Primary;
 
                         for (final SAMRecord supp : supplementals) {
-                            final SAMRecord out = clone(sourceRec);
+                            final SAMRecord out = sourceRec.deepCopy();
```

Comment:
perhaps it would be better to do a deep copy at the step before (and if) the bases get reverse complemented? is that possible?